src/pke/lib/schemebase/base-multiparty.cpp

```diff
@@ -424,6 +424,68 @@ std::shared_ptr<std::map<usint, EvalKey<Element>>> MultipartyBase<Element>::Mult
     return EvalKeyMapSum;
 }
 
+template <class Element>
+Ciphertext<Element> MultipartyBase<Element>::IntMPBootAdjustScale(ConstCiphertext<Element> ciphertext) const {
+#if NATIVEINT != 64
+    std::string errmsg = "Interactive Multiparty Bootstrapping for CKKS is only supported with NATIVEINT = 64";
+    OPENFHE_THROW(not_available_error, errmsg);
+#endif
+
+    std::string msg = "IntMPBootAdjustScale: supports only DCRTPoly as input";
```

Comment:
This error may not tell a user what you exactly mean as Element may be DCRTPoly. This comment applies to all added exception error messages in this file

Reply:
Fixed as suggested